### PR TITLE
[DO NOT MERGE] Edited Getting Started.

### PR DESCRIPTION
**Pull request by felicianotech**
Example pull request.

### CODE REVIEW

**Comment by felicianotech** [comment]
This is fake, why would I merge this?

File: docs/getting-started.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ published: 'Sunday, July 19th, 2009'
 title: Getting Started
 ---
 
-Congratulations on selecting Linode as your cloud hosting provider! We're glad you've decided to join our community, and we're excited to help you get started with your first Linode server. This guide will help you sign up for an account, deploy a Linux distribution, boot your Linode, and perform some basic system administration tasks. By the time you reach the end, you'll have a Linode up and running in the data center of your choice.
+Hi, welcome to on selecting Linode as your cloud hosting provider! We're glad you've decided to join our community, and we're excited to help you get started with your first Linode server. This guide will help you sign up for an account, deploy a Linux distribution, boot your Linode, and perform some basic system administration tasks. By the time you reach the end, you'll have a Linode up and running in the data center of your choice.
```
Comment (felicianotech) [inline docs/getting-started.md]: Why did you change this line?

**Comment by alexfornuto** [comment]
http://krl.io/ba58d
